### DISCUSSION TITLE
Set default debounce value

### DISCUSSION THREAD

**Discussion by ghost** [livewire/livewire#1121] [3. Feature ideas]
Default the debounce is 150ms and it's possible to change the debounce per input:

`<input type="text" wire:model.debounce.1000ms="name">` 

It would be great if it's possible to set the default debounce value.

**Comment by ChrisSantiago82** [comment]
I was just looking for the same thing. Maybe this could be added to the livewire config?

**Comment by joshhanley** [comment]
@Vincent-Ntriga and @ChrisSantiago82 yeah I can understand how that'd be helpful!

One issue we have with that is the config is in PHP and it would have to be somehow sent across to Javascript for the default to be set on the inputs and used, which we currently don't have a solution for doing.

I've changed this discussion to be a feature request 🙂
